src/node/entry.ts

```diff
@@ -32,8 +32,18 @@ const version = pkg.version || "development"
 const commit = pkg.commit || "development"
 
 const main = async (args: Args): Promise<void> => {
+  args = await readConfigFile(args)
```

Comment:
This might be dumb but what do you think about calling this before calling `main`?
```
wrap(async () => main(await readConfigFile(args)))
```

Reply:
I think it adds a lot of verbosity versus doing it right in main.

Reply:
I'm probably thinking in a sort of functional way where it would be odd for the function I'm calling to modify the arguments I give it. No big deal either way, was just curious what your thoughts were.

Reply:
I completely agree with the sentiment but I like keeping everything promise related in `main` just so it's easier to read.